promql/quantile.go

```diff
@@ -370,18 +372,22 @@ func coalesceBuckets(buckets buckets) buckets {
 //
 // As a somewhat hacky solution until ingestion is atomic per scrape, we
 // calculate the "envelope" of the histogram buckets, essentially removing
-// any decreases in the count between successive buckets.
+// any decreases in the count between successive buckets. We return a bool
+// to indicate if this monotonicity was forced or not.
 
```

Comment:
This blank line has to go. (Was already wrong before, but now that we are on it…)